app/business_model/forms.py

```diff
@@ -67,3 +67,32 @@ def clean(self):
         else:
             raise ValidationError("This form cannot be blank")
         return cleaned_data
+
+
+class EquityDataForm(forms.ModelForm):
+    class Meta:
+        model = EquityData
+        exclude = ["scenario", "debt_start"]
+
+    def __init__(self, *args, **kwargs):
+        include_shs = kwargs.pop("include_shs", False)
+        super().__init__(*args, **kwargs)
+
+        if not include_shs:
+            for field in self.fields:
+                if "SHS" in field:
+                    self.fields[field].widget = forms.HiddenInput()
+                    self.fields[field].required = False
+
+        labels = {
```

Comment:
If you set this in the Meta class it does replace them automatically, see projects/forms.py::ScenarioCreateForm for example. I find even more elegant to define them directly in the model definition with the [verbose_name](https://docs.djangoproject.com/en/4.2/ref/models/fields/#verbose-name) argument ^^